django/db/models/fields/__init__.py

```diff
@@ -828,18 +828,17 @@ def get_flatchoices(self, include_blank=True,
         first_choice = blank_choice if include_blank else []
         return first_choice + list(self.flatchoices)
 
+    @warn_about_renamed_method('Field', '_get_val_from_obj', 'value_from_object',
+                               RemovedInDjango21Warning)
     def _get_val_from_obj(self, obj):
-        if obj is not None:
```

Comment:
If it's not clear, my suggestion is merely to leave the original implementation of `_get_val_from_obj()` instead of changing it to `return self.value_from_object(obj)` and to change the deprecation warning/docs to avoid implying that the methods are completely identical in their behavior. I can finish this if you have given up on it.